src/utils/dictionaryValuesFromArrToString.js

```diff
@@ -0,0 +1,8 @@
+
+export default function dictionaryValuesFromArrToString(originalDictionary) {
+  let dictionary = {}
+  for (const [key, value] of Object.entries(originalDictionary)) {
+    dictionary[key] = [].concat(value).join(",")
+  }
+  return dictionary;
+}
```

Comment:
This project uses [rambda](https://selfrefactor.github.io/rambda/#/) as its utility library. This can be expressed as:

```javascript
import {map} from 'rambda'

const dictionaryValuesFromArrToString = map(x => [].concat(x).join(','))
```

Also the name is quite long, and it could be changed to highlight its concrete purpose eg `serializeParams`.

Regarding naming:
 - "dictionary" is python terminology, use "object" in javascript
 - by convention, most words are never shortened in function names in javascript (and "Array" is not part of that limited list)